code/_onclick/hud/action.dm

```diff
@@ -1,8 +1,3 @@
-#define AB_ITEM 1
-#define AB_SPELL 2
-#define AB_INNATE 3
-#define AB_GENERIC 4
-
```

Comment:
Another pull had stopped the use of these defines, but they had forgotten to remove them here